build/deploy.js

```diff
@@ -24,20 +24,30 @@ const deployToFirebase = (cb) => {
     return
   }
 
+  debug('Installing firebase-tools...')
+
+  exec('npm install -g firebase-tools', (error, stdout) => {
```

Comment:
My thinking was that firebase-tools does not have to be installed when not deploying. Currently, it installs even when the deploy script then doesn't use it. Should be a non-issue if we use Travis Firebase deploy.